guilds_tree.go

```diff
@@ -75,6 +75,17 @@ func (gt *GuildsTree) createGuildNode(n *tview.TreeNode, g discord.Guild) {
 
 func (gt *GuildsTree) channelToString(c discord.Channel) string {
 	var s string
+	tag := "[::d]"
+	read_states := discordState.Ready().ReadyEventExtras.ReadStates
+
+	for i := range read_states {
+		if read_states[i].ChannelID == c.ID {
+			if read_states[i].LastMessageID != c.LastMessageID {
+				tag = "[::bi]"
+			}
+		}
+	}
```

Comment:
You could do
```go
if read_states[i].ChannelID == c.ID && read_states[i].LastMessageID != c.LastMessageID {
	tag = "[::bi]"
}
```
instead of:
```go
if read_states[i].ChannelID == c.ID {
	if read_states[i].LastMessageID != c.LastMessageID {
		tag = "[::bi]"
	}
}
```